consumer.go

```diff
@@ -591,6 +592,10 @@ func (child *partitionConsumer) parseResponse(response *FetchResponse) ([]*Consu
 				child.offset++ // skip this one so we can keep processing future messages
 			} else {
 				child.fetchSize *= 2
+				// int32 overflow
```

Comment:
Can you please change the comment to `check int32 overflow` or remove it?

Reply:
sure, let me update the comment